core/src/stories/Migration/Migration.stories.mdx

```diff
@@ -20,9 +18,18 @@ Previously we have not had a set naming convention for our events. This has led
 components where some events has had a `custom` prefix, and others have had other ways of indicating whether
 or not it is a custom event. We have now settled on a naming convention for all events.
 Both **external**, events we expect you as a user to interact with, and **internal**, events that are used within
-our components to update state/react to changes.
+our components to update state/react to changes. 
 
-In short:
+## Prefix change
+Furthermore we decided to rename the previously used 'sdds'-prefix to 'tds'. 
+This occurred as a result of a deliberate choice to facilitate the transition to the new Tegel package and prevent conflicts with previously used components.
+
+In short, before the prefix change:
```

Comment:
```suggestion
For example, before the prefix change:
```
I think this works better, because this is just a few examples, not a short description of all the changes.